en/ui-system/components/editor/ui-transform.md

```diff
@@ -4,6 +4,8 @@ The UITransform component defines the rectangle information on the UI, including
 
 Click the __Add Component__ button at the bottom of the __Inspector__ panel and select __UITransform__ from __UI__ to add the UITransform component to the node.
 
+Api can refer to [UITransform API](https://docs.cocos.com/creator3d/api/en/classes/ui.uitransform.html).
```

Comment:
Api can refer to [UITransform API](https://docs.cocos.com/creator3d/api/en/classes/ui.uitransform.html).
-->
Please refer to the [UITransform API](https://docs.cocos.com/creator3d/api/en/classes/ui.uitransform.html).